external-libraries.md

```diff
@@ -99,13 +99,6 @@ URL:     https://github.com/lemire/javaewah
 License: Apache-2.0
 ```
 
-```yaml
```

Comment:
Shouldn't we add the library there, too? (not important for this PR, will be done in the release process)

Reply:
Yes, this is still a mess. Most importantly, we can get rid of the annoying config for jfoenix.